### PR TITLE
 Add Spanish translation

### PR DESCRIPTION
**Pull request by t4x0n**

### CODE REVIEW

File: locale/es_ES/locale.xml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+<!DOCTYPE locale SYSTEM "../../../../lib/pkp/dtd/locale.dtd">
+
+<!--
+  * plugins/themes/defaultManuscript/locale/es_ES/locale.xml
+  *
+  * Copyright (c) 2014-2017 Simon Fraser University
+  * Copyright (c) 2003-2017 John Willinsky
+  * Distributed under the GNU GPL v2. For full terms see the file docs/COPYING.
+  *
+  * Localization strings.
```
Comment (marcbria) [inline locale/es_ES/locale.xml]: We label the files that are not entirely translated by our team. Please, modify this line to:
"* Localization strings (contrib)."

File: locale/es_ES/locale.xml [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  -->
+
+<locale name="es_ES" full_name="Español (España)">
+	<message key="plugins.themes.defaultManuscript.name">Manuscrito (Tema hijo del tema predeterminado)</message>
```
Comment (marcbria) [inline locale/es_ES/locale.xml]: As the theme own name, we think it shouldn't be translated.
Avoid repeating the same word in the same line, with synonyms or rephrasing. In this case, it's clear it's a theme so we suggest this:
"Manuscript (hijo del tema predeterminado)"

File: locale/es_ES/locale.xml [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+	<message key="plugins.themes.defaultManuscript.description">Un tema limpio y sencillo con un diseño en caja que imita un documento en papel.</message>
+	<message key="plugins.themes.defaultManuscript.option.typography.montserrat_notoSerif">Montserrat/Noto Serif</message>
+	<message key="plugins.themes.defaultManuscript.option.accentColour.label">Color de realce</message>
```
Comment (marcbria) [inline locale/es_ES/locale.xml]: Dado el contexto, mejor "destacado" que "realzado":
"Color de destacados"

**Comment by NateWr** [comment]
@t4x0n Thanks for the translation! @marcbria handles the Spanish translations for OJS, so he's the one who makes final decisions on matters of grammar and style in the translations.

If you could update the PR with his recommended changes, that'd be great! :+1:

**Comment by t4x0n** [comment]
Yes @marcbria, I want to help you with the Spanish translation and thank you!  Please send me your glossary to improve my translations

**Pull request merged**